batchspawner/batchspawner.py

```diff
@@ -499,28 +522,18 @@ def user_env(self, env):
         return env
 
     def get_env(self):
-        """Add user environment variables"""
+        """Add user environment variables.
+
+        Everything here should be passed to the user's job.  If it is
```

Comment:
All user environment variables will be passed to the user's job.

Caution: If these variables are used for authentication to the batch system commands as an admin,
be aware that the user will receive access to these as well.

Reply:
Thanks!

This had made me realize one possible solution is to separate req_keepvars and get_env more.  Right now they are basically the same, but by using them more cleverly we can separate admin variables and user variables.  It's already possible in fact, but not very convenient.  Will comment in the other security tracking issue.